<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitnessCard</class>
 <widget class="QMainWindow" name="FitnessCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FitnessCard</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="exercisesTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>554</width>
      <height>244</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="viewExercisesTab">
     <attribute name="title">
      <string>View Exercises</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QComboBox" name="viewSelect">
          <item>
           <property name="text">
            <string>1 week</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 weeks</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 month</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2 months</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3 months</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6 months</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1 year</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="viewLabel">
          <property name="text">
           <string>View length:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="exerciseTable"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="addExerciseTab">
     <attribute name="title">
      <string>Add exercise</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="typeLabel">
          <property name="text">
           <string>Exercise type:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="typeSelect">
          <item>
           <property name="text">
            <string>Running</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Skiing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bicycling</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Badminton</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tennis</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Swimming</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Walking</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lengthLabel">
          <property name="text">
           <string>Length:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lengthLine"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="intensityLabel">
          <property name="text">
           <string>Intensity:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="intensitySelect">
          <item>
           <property name="text">
            <string>High</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Low</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
